code/game/objects/items/storage/belt.dm

```diff
@@ -130,6 +131,7 @@
 	name = "medical belt"
 	desc = "Can hold various medical equipment."
 	icon_state = "medical"
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
```suggestion

```
It should be fine to make the medical belt bulky, don't be afraid of backlash. ~~unless thats your medical bias showing~~
Otherwise good change

Reply:
Actually, unlike the other belts, security, engineering and assault, there already is an item that holds pretty much exactly the same stuff as the medical belt, but better. The First Aid Kit. Also, unlike the CE and HOS that both have belts for their job, the CMO has two belts. The CMO's surgery kit and the Belt defib.
You would have to drop the belt to use the defibrillator in both hands. I will go back soon and specifically give the CMO their own belt with sprites and all, dw

Reply:
Also the main issue right now is VT. They need a belt. I will replace with a tools pouch that holds any 4 assortment of tools in a smaller package that can go on your pocket slot.

Reply:
Ah, that makes sense. A Eris/TGMC style pouch system  that can fit in your pocket but not your bag would be very neat